engine/src/main/java/org/hibernate/validator/constraints/ScriptAssert.java

```diff
@@ -111,6 +111,11 @@
 	String alias() default "_this";
 
 	/**
+	 * @return The name of the field on which you would like to report a validation error for.
```

Comment:
"field" = "property". In general, BV abstracts from fields/getters and exposes them as properties in a unified manner. Also the default behavior should be mentioned. Something along these lines:

> If given, the resulting constraint violation will be reported on the specified property. If not given, the constraint violation will be reported on the annotated bean.

Ah, and could you add "@since 5.4"?